doc/classes/ResourceImporterLayeredTexture.xml

```diff
@@ -56,7 +56,9 @@
 			It's recommended to enable mipmaps in 3D. However, in 2D, this should only be enabled if your project visibly benefits from having mipmaps enabled. If the camera never zooms out significantly, there won't be a benefit to enabling mipmaps but memory usage will increase.
 		</member>
 		<member name="mipmaps/limit" type="int" setter="" getter="" default="-1">
-			Unimplemented. This currently has no effect when changed.
+			Sets the maximum number of mipmaps for a texture. The smaller the limit, the sharper the texture will appear from a distance. This can help alleviate the overblurring of distant images, especially ones at a low resolution.
+			If set to [code]-1[/code], the full mipmap chain will be generated. If [code]0[/code], no mipmaps will be generated.
```

Comment:
Similar sentiment as before regarding mipmap chains.
```suggestion
			If [code]-1[/code], generates as many mipmaps as possible. If [code]0[/code], no mipmaps will be generated.
```
Also applies to the other descriptions for this.